pkg/cmd/get/cluster/cmd.go

```diff
@@ -49,6 +49,7 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 	}
 
 	cmd.Flags().StringVar(&o.Clusterset, "clusterset", "", "ClusterSet of the clusters")
+	cmd.Flags().StringVarP(&o.output, "output", "o", "tree", "output format can be tree, table or yaml")
```

Comment:
I think it is better you have a common PrinterOption and addFlags here.

You can initialize the printer with what real tree printer or table printer wants to do here.